src/ctia/http/routes/exploit_target.clj

```diff
@@ -69,16 +74,19 @@
                 :header-params [api_key :- (s/maybe s/Str)]
                 :summary "List ExploitTarget by external id"
                 :capabilities #{:read-exploit-target :external-id}
+                :middleware [wrap-un-store wrap-cache-control]
                 (paginated-ok
                  (page-with-long-id
                   (read-store :exploit-target list-exploit-targets
                               {:external_ids external_id} q))))
+
            (GET "/search" []
                 :return (s/maybe [StoredExploitTarget])
```

Comment:
I think that we should define a new schema, we have `New*` and `Stored*` schemas, I think that in this case it doesn't return nor a `NewExploitTarget` or a `StoredExploitTarget` but just an `ExploitTarget`